### ISSUE TITLE
Output more Info about Plattform and Browsername

### ISSUE THREAD

**Issue by kuddl** [dalekjs/dalek-reporter-html#9]
It would be great, if the "name" of the browser would be reflected in the HTML output.

Like:
```
 "ie10win7": <<<<=== this name should be reflected 
 {  
      "type":"remote",
      "platform": "Win 7",
      "version": "10",
      "actAs":"ie",
      "host": "192.168.79.130",
      "port":"9020"
    },
```
and output should be like this... 

![dalek test_2015-01-09_18-41-38](https://cloud.githubusercontent.com/assets/127226/5684294/3a6c6ed8-982f-11e4-9edd-432477b16924.png)

I tried to find the appropriate code, but couldn't find where to get the Info from. Like which event, i should bind.... 

This would help a lot, if you test a lot of different Versions of IE on different VMs.
And since I kinda name my Browsers in the Dalekfile.json with some hints about Version and Plattform, this would help a lot!

Thanks

**Comment by asciidisco** [comment]
Yes, that is a big improvement. Do you think you could bring in a PR for this (I´m currently very busy with the DalekJS rewrite & getting out the new version - so I probably won't have time for this)
